Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/statecheck/DccStateChecker.kt

```diff
@@ -31,15 +33,17 @@ class DccStateChecker @Inject constructor(
         dscRepository.dscData
     ) { appConfig, dscData ->
 
+        // TODO check revoked state
```

Comment:
You can add a line here to test the newly added state 
```kotlin
return@combine Revoked
```

moreover you can test Blocked and other states if you want , Please see the text message for DE and  other countries